locales/en.yml

```diff
@@ -21,27 +41,162 @@ pico:
+    error:
+      asExitCodeAndJson: "An error occured when checking for updates. See the JSON output for more details."
+    info:
+      checking: "Checking for updates..."
+      noUpdates: "No updates available."
+      updatesAvailable: "Updates available."
+  sys-upgrade:
+    description: "Upgrade the system"
+    error:
+      updating: "An error occured when updating the system."
+    info:
+      updating: "Updating the system..."
+    now: "Trigger a system upgrade now"
+
+  tasks:
+    description:  "Create and manage tasks"
+    stacks:
+      new:
+        error:
+          emptyCommand: "The command cannot be empty."
+          emptyDescription: "The description cannot be empty."
+          emptyName: "The name cannot be empty."
+          noCommand: "No command specified."
+          noDescription: "No description specified."
+          noName: "No name specified."
+        info:
+          askCommand: "Command: "
+          askDescription: "Description: "
+          askName: "Name: "
+          taskCreated: "Task has been created successfully."
+    list:
+      description: "List all tasks"
+      noTasks: "No tasks found."
+      info:
+        foundTasks: "Found %d tasks."
+      options:
+        json:
+          description: "output the tasks in JSON format"
+    new:
+      description: "Create a new task"
+      options:
+        assumeYes:
+          description: "assume yes for all prompts"
+        at:
+          description: "schedule the task to execute at a specific time (hh:mm)"
+        command:
+          description: "specify the command to execute"
+        description:
+          description: "specify the task's description"
+        every:
+          description: "schedule the task to execute every X time (minutes, hours, days)"
+        name:
+          description: "specify the task's name"
+        needConfirm:
+          description: "ask for confirmation before executing the task"
+        onBattery:
+          description: "execute the task only when the system is on battery"
+        onCharge:
+          description: "execute the task only when the system is charging"
+        onConditionCommand:
+          description: "execute the task on condition command"
+        onCpuTemp:
+          description: "execute the task when CPU temperature is higher than 60°C"
+        onCpuUsage:
+          description: "execute the task when CPU usage is higher than X%"
+        onDeviceConnect:
+          description: "execute the task when a device is connected"
+        onDeviceDisconnect:
+          description: "execute the task when a device is disconnected"
+        onDisconnect:
+          description: "execute the task when the system is disconnected from the network"
+        onFullBattery:
+          description: "execute the task when the system is fully charged"
+        onHighCpuUsage:
+          description: "execute the task when CPU usage is higher than X%"
```

Comment:
This is the same string as `onCpuUsage`. Are both of them necessary?

Reply:
Maybe here it shouldn't be `XX%` but a pre-defined number.

Reply:
> This is the same string as `onCpuUsage`. Are both of them necessary?

Not sure tbh, I referred to [`create-task.go` from Kinetic branch](https://github.com/Vanilla-OS/vanilla-system-operator/blob/kinetic/cmd/create-task.go) where there were 2 separate entries.

Reply:
`onHighCpuUsage` in the kinetic branch is "higher than 50%", and `onCpuUsage` is "higher than XX%". Both of then are "higher than XX%" here. I think `onHighCpuUsage` should be "execute the task when CPU usage is higher than 50%"

Reply:
```suggestion
          description: "execute the task when CPU usage is higher than 50%"
```